ibm_vpd_utils.cpp

```diff
@@ -371,29 +369,10 @@ vpdType vpdTypeCheck(const Binary& vpdVector)
         // KEYWORD VPD FORMAT
         return vpdType::KEYWORD_VPD;
     }
-    else if (((vpdVector[SPD_BYTE_3] & SPD_BYTE_BIT_0_3_MASK) ==
-              SPD_MODULE_TYPE_DDIMM) &&
-             (is11SFormat.compare(MEMORY_VPD_START_TAG)))
+    else if (is11SFormat.compare(MEMORY_VPD_START_TAG) == 0)
     {
-        // DDIMM Memory VPD format
```

Comment:
am not following why these ddimm changes are removed as a part of this commit. can you please give a gerrit commit to review with ease